src/Mailer/Email.php

```diff
@@ -1484,6 +1485,17 @@ protected function _encode($text)
     }
 
     /**
+     * Decode the specified string
+     *
+     * @param string $text String to decode
+     * @return string Decoded string
+     */
+    protected function _decode($text)
+    {
+        return mb_decode_mimeheader($text);
```

Comment:
I appreciate your thoroughness in writing tests, but the tests are primarily ensuring PHP still works.